services/postgres/migrations/20240129140329_init/migration.sql

```diff
@@ -0,0 +1,183 @@
+-- CreateEnum
+CREATE TYPE "TransactionType" AS ENUM ('CREDIT', 'DEBIT');
+
+-- CreateTable
+CREATE TABLE "Org" (
+    "id" TEXT NOT NULL,
```

Comment:
This might be nitpicky, but is it best to use text for these type of short string fields? I know I used to do VARCHAR(32) or something in the past. Does "text" not have any performance hit? I know this is generated, but I assume you can give prisma a hint.